docs/hardware.rst

```diff
@@ -408,13 +408,26 @@ To assist you, we have compiled the following partial list of
-| Model                   | Testing Date   | Tails Versions   | Price (new)        | Price (used)       | Notes                                                                                                                                                       |
-+=========================+================+==================+====================+====================+=============================================================================================================================================================+
-| HP LaserJet 400 M401n   | 06/2015        | 1.4              | $178.60 (Amazon)   | $115.00 (Amazon)   | Monochrome laser printer. Heavy (10 lbs.) When adding the printer in Tails, you need to set "Make and model" to "HP LaserJet 400 CUPS+Gutenprint v5.2.9".   |
-+-------------------------+----------------+------------------+--------------------+--------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| HP Deskjet 6940         | 04/2015        | 1.3.2            | $639.99 (Amazon)   | $196.99 (Amazon)   | Monochrome Inkjet printer                                                                                                                                   |
-+-------------------------+----------------+------------------+--------------------+--------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------+
++-------------------------+--------------+----------------+--------------------+
+| Printer Model           | Testing Date | Tails Versions | Printer Type       |
++=========================+==============+================+====================+
+| HP Deskjet F4200        | 06/2017      | 3.0            | Color Inkjet       |
++-------------------------+--------------+----------------+--------------------+
+| HP Deskjet 1112         | 06/2017      | 3.0            | Color Inkjet       |
++-------------------------+--------------+----------------+--------------------+
+| HP LaserJet 400 M401n   | 06/2015      | 1.4            | Monochrome Laser   |
++-------------------------+--------------+----------------+--------------------+
+| HP Deskjet 6940         | 04/2015      | 1.3.2          | Monochrome Injket  |
++-------------------------+--------------+----------------+--------------------+
+
+.. note:: We've documented both the HP Deskjet F4200 and HP LaserJet 400 M401n
+         with screenshots of the installation process, in our section on
+         `setting up printers in Tails
+         <https://docs.securedrop.org/en/stable/tails_printing_guide.html>`_.
```

Comment:
@justintroutman Yup, I would like to do some cleanup of the reStructuredText style conventions used throughout our documentation, and I'm sorry that the current lack of consistency was confusing for you. I'll file an issue with some notes about what stylistic conventions I think we should adopt going forward, and we can address this holistically after the 0.4 release.